src/components/Forum/ForumMain/ForumMainPage.js

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from "react";
+import { useOutletContext } from "react-router-dom";
+import Select from "react-select";
+import axios from "axios";
+import SectionColumn from "./SectionColumn";
+import Loading from "../../Common/Loading";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const sortOption = [
+  { label: "Category", value: "name" },
+  { label: "Popular Section", value: "popularSection" },
+  { label: "Newest Post", value: "newestPost" },
+  { label: "Newest Comment", value: "newestComment" },
+];
+
+export default function ForumMainPage() {
+  const [, setErrorMessage] = useOutletContext();
+  const [categoryList, setCategoryList] = useState([]);
+  const [sort, setSort] = useState("name");
+  const { getAccessTokenSilently } = useAuth0();
+
+  useEffect(() => {
+    const getCategoryList = async () => {
+      try {
+        const token = await getAccessTokenSilently();
+        const { data } = await axios.get(
+          `${process.env.REACT_APP_BACKEND_URL}/categories/all/sort/${sort}`,
+          {
+            headers: {
+              Authorization: `Bearer ${token}`,
+            },
+          }
+        );
+        setCategoryList(data);
+      } catch (error) {
+        setErrorMessage(error.message);
+      }
+    };
+    getCategoryList();
+  }, [setErrorMessage, sort, getAccessTokenSilently]);
+
+  const sectionList = !!categoryList.length ? (
```

Comment:
```suggestion
  const sectionList = categoryList.length ? (
```

Same thing. 0 is falsy, > 0 is truthy.